packages/trader/src/App/Containers/RealAccountSignup/address-details.jsx

```diff
@@ -107,7 +108,7 @@ class AddressDetails extends Component {
                                             />
                                             <fieldset className='address-state__fieldset'>
                                                 <Dropdown
-                                                    is_alignment_top={(window.innerHeight < 930)}
+                                                    // is_alignment_top={(window.innerHeight < 930)}
```

Comment:
If this isn't needed, maybe it can be removed instead of commented? I added it previously to compensate for lack of space below `Dropdown` to display the `list`.

Reply:
sure